tests/moe/test_unified_moe_b12x.py

```
@@ -0,0 +1,760 @@
+            intermediate_size=512,
+            num_experts=256,
+            top_k=2,
+            source_format=source_format,
+        )
+        unified = _run_b12x_unified(layer, act_pack, weight_pack).clone()
+        w1_alpha = tensors["w1_alpha"]
+        w2_alpha = tensors["w2_alpha"]
+        if source_format == "compressed_tensors":
+            w1_alpha = w1_alpha.reciprocal()
+            w2_alpha = w2_alpha.reciprocal()
+        legacy = wrapper.run(
```

Ruff (0.15.21)

[error] Undefined name `wrapper`

(F821)